### DISCUSSION TITLE
[Feature request] Input Date: Web

### DISCUSSION THREAD

**Discussion by CarmenchUX** [ing-bank/lion#1844] [General]
https://lion-web.netlify.app/components/input-date/overview/

We need the Date input to allow setting it for just the month and year. We have cases where we need to know a date but we don't ask for the specific day (asking for the day causes a lot of blocks by users) especially when the date is for fields related to their employment status. Example: When did you start working for that company?
 
Can it be parameterized? It would be very useful to be able to choose what parameters we need.

**Comment by tlouisse** [comment]
Hi, it would be nice to have these at some point:
- `LionInputDateTime` (extends LionInputDate)
- `LionInputDateTimePicker` (extends LionInputDatepicker and uses DateTime functionality as mixin)

For now, you could maybe use the second example in this tutorial (wrapper around native input[type=datetime]):
https://lion-web.netlify.app/guides/how-to/extend-a-native-input/
